packages/x-date-pickers-pro/src/NextDateRangePicker/describes.NextDateRangePicker.test.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { describeConformance, screen, userEvent } from '@mui/monorepo/test/utils';
+import { Unstable_NextDateRangePicker as NextDateRangePicker } from '@mui/x-date-pickers-pro/NextDateRangePicker';
+import { describeRangeValidation } from '@mui/x-date-pickers-pro/tests/describeRangeValidation';
+import { describeValue } from '@mui/x-date-pickers/tests/describeValue';
+import {
+  adapterToUse,
+  buildFieldInteractions,
+  createPickerRenderer,
+  expectInputValue,
+  wrapPickerMount,
+} from 'test/utils/pickers-utils';
+
+describe('<NextDateRangePicker /> - Describes', () => {
+  const { render, clock } = createPickerRenderer({ clock: 'fake' });
+
+  const { clickOnInput } = buildFieldInteractions({ clock });
+
+  describeConformance(<NextDateRangePicker />, () => ({
+    classes: {},
+    muiName: 'MuiDateRangePicker',
+    wrapMount: wrapPickerMount,
+    refInstanceof: window.HTMLDivElement,
+    skip: [
+      'componentProp',
+      'componentsProp',
+      'themeDefaultProps',
+      'themeStyleOverrides',
+      'themeVariants',
+      'mergeClassName',
+      'propsSpread',
+      'rootClass',
+      'reactTestRenderer',
+    ],
+  }));
+
+  describeRangeValidation(NextDateRangePicker, () => ({
+    render,
+    clock,
+    componentFamily: 'new-picker',
+    views: ['day'],
+  }));
+
+  describeValue(NextDateRangePicker, () => ({
```

Comment:
In my PR, I only added the `describeValue` on `MobileXXX` and `DesktopXXX`, not on the responsive one

I think it's worth adding it at least to those two.
And for the responsive one I don't think it's worth it 

Did you have an issue writing the mobile version of the range `describeValue` ?